scripts/components/SongCards.js

```diff
@@ -101,7 +101,7 @@ class SongCards extends Component {
 
     renderSongs(start, end) {
         const chunk = 5;
-        const {authed, dispatch, playlist, playlists, playingSongId, songs, users} = this.props;
+        const {authed, dispatch, playlist, playlists, playingSongId, songs, users, toggleStats} = this.props;
```

Comment:
I assumed you have to pass props from the container down to the final child, might have been wrong about that.